manifests/quick-start/fluentd-forward.yaml

```diff
@@ -11,7 +11,7 @@ spec:
         bind: 0.0.0.0
         port: 24224
   replicas: 1
-  image: zhu733756/fluentd:config-reloader
+  image: kubesphere/fluentd:v1.14.4-amd64
```

Comment:
Maybe `fluentd:v1.14.4-operator` is better because this image will finally migrate to fluent org and there should already have fluentd:v1.14.4